data_specification/data_specification_generator.py

```diff
@@ -765,7 +765,8 @@ def write_structure(
                 _Field.IMMEDIATE: structure_id})
             cmd_string += "repeats=reg[{0:d}]".format(repeats)
         else:
-            _bounds(Commands.WRITE_STRUCT, "repeats", repeats, 0, 16)
+            _bounds(Commands.WRITE_STRUCT, "repeats",
+                    repeats, 0, MAX_STRUCT_SLOTS)
```

Comment:
Ah, I think this is another one that isn't really needed (though again, not really used anyway)

Reply:
Actually, looking at it, this is a good change as it makes a magic number into a constant